src/runtime/internal/dom.ts

```diff
@@ -148,7 +148,7 @@ export function get_root_for_style(node: Node): ShadowRoot | Document {
 	if (!node) return document;
 
 	const root = node.getRootNode ? node.getRootNode() : node.ownerDocument;
-	if ((root as ShadowRoot).host) {
+	if (root && (root as ShadowRoot).host) {
```

Comment:
What does this change intended to address? This seems unrelated to the `src` attribute handling mentioned in #6575.

Reply:
The return of `node.ownerDocument` is possibly null.
And in the case of Svelte Native returns null at least in this issue #6675.
Therefore it needs to check nullability.